devel/openssl10/Portfile

```diff
@@ -0,0 +1,128 @@
+                    http://artfiles.org/openssl.org/source/ \
+                    ftp://ftp.linux.hr/pub/openssl/source/ \
+                    ftp://guest.kuria.katowice.pl/pub/openssl/source/
+
+checksums           sha1    b9aec1fa5cedcfa433aed37c8fe06b0ab0ce748d \
+                    rmd160  f268c8f87ee6b8ca1523761b064de575f6851ae0 \
+                    sha256  ae51d08bba8a83958e894946f15303ff894d75c2b8bbd44a852b64e3fe11d0d6 \
+                    size    5348369
+
+patchfiles          install-headers-HFS+.patch \
+                    parallel-building.patch \
+                    remove-duplicate-bn_print-doc.patch
+
+configure.ccache    no
+configure.perl      /usr/bin/perl
+configure.cmd       ./Configure
+configure.args     -L${prefix}/lib \
+                    no-krb5 \
+                  --openssldir=${prefix}/etc/openssl \
+                  --libdir=lib/openssl-1.0 \
+                    shared \
+                    zlib
+# Use SDK if necessary.
+if {${configure.sdkroot} ne ""} {
+    configure.args-append   '-isysroot ${configure.sdkroot}' \
+                            -Wl,-syslibroot,${configure.sdkroot}
+}
+
+set merger_arch_compiler no
+array set merger_configure_args {
+    ppc     darwin-ppc-cc
+    i386    darwin-i386-cc
+    ppc64   darwin64-ppc-cc
+    x86_64  darwin64-x86_64-cc
+}
+platform darwin {
+    # Don't use i386 assembly on Tiger (#38015, #43303).
+    if {${os.major} <= 8} {
+        append merger_configure_args(i386) { no-asm}
+    }
+    # Don't use x86-64 assembly on Tiger or Leopard.
+    if {${os.major} <= 9} {
+        append merger_configure_args(x86_64) { no-asm}
+    }
+}
+# Don't pass --host to configure.
+array set merger_host {ppc {} i386 {} ppc64 {} x86_64 {}}
+
+if {![variant_isset universal]
+        && [info exists merger_configure_args(${configure.build_arch})]} {
+    configure.args-append $merger_configure_args(${configure.build_arch})
+}
+configure.universal_args-delete --disable-dependency-tracking
+
+# Parallel builds don't quite work (#46719).
+use_parallel_build  no
+
+test.run            yes
+
+if {[variant_isset universal]} {
+    pre-destroot {
+        global merger_dont_diff
+        if {[llength ${universal_archs_to_use}] > 2} {
+            lappend merger_dont_diff ${prefix}/include/openssl/opensslconf.h
+        }
+    }
+}
+
+post-destroot {
+    # Move some files around
+    xinstall -d ${destroot}${prefix}/include/openssl-1.0
+    move ${destroot}${prefix}/include/openssl ${destroot}${prefix}/include/openssl-1.0/
+    move ${destroot}${prefix}/bin/openssl ${destroot}${prefix}/bin/openssl-1.0
+
+    # Update includedir in .pc files
+    reinplace "s|/include$|/include/openssl-1.0|" {*}[glob ${destroot}${prefix}/lib/openssl-1.0/pkgconfig/*.pc]
+
+    file delete -force ${destroot}${prefix}/etc
+    file delete ${destroot}${prefix}/bin/c_rehash
+}
+
+destroot.args       MANDIR=${prefix}/share/man MANSUFFIX=ssl
```

Comment:
Won't the man pages conflict with the openssl port?

Reply:
Never mind, I see that's taken care of by the destroot.target. I guess there's no need for these args though.

Reply:
Thank you! This line is now removed.